src/components/lineup/delineate.ts

```diff
@@ -0,0 +1,56 @@
+import { groupBy } from 'lodash'
+import moment from 'moment'
+
+import { LineupItem } from './types'
+
+const NOW = moment()
+const START_OF_DAY = moment(NOW).startOf('day')
+const START_OF_YESTERDAY = moment(NOW).subtract(1, 'days').startOf('day')
+const START_OF_WEEK = moment(NOW).startOf('week')
+const START_OF_LAST_WEEK = moment(NOW).subtract(1, 'week').startOf('week')
+const START_OF_MONTH = moment(NOW).startOf('month')
+
+const Delineations = Object.freeze({
```

Comment:
Wonder if we could share this code b/w web & mobile at some point (maybe when we rewrite web lineup)

Reply:
Yes for sure, I kept it separate for now because the web implementation is pretty different. But we can consolidate in the future